src/Neo/Persistence/MemoryStoreProvider.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Neo.Persistence
 {
-    internal class MemoryStoreProvider : IStoreProvider
+    public class MemoryStoreProvider : IStoreProvider
```

Comment:
@Jim8y Now that we have two constructors in neoSystem, we need to expose also de provider, otherwise, devpack also fails.